src/main/scala/com/databricks/labs/overwatch/env/Workspace.scala

```diff
@@ -130,6 +130,73 @@ class Workspace(config: Config) extends SparkSessionWrapper {
     ApiCallV2(config.apiEnv, workspaceEndpoint).execute().asDF().withColumn("organization_id", lit(config.organizationId))
   }
 
+  /**
+   * Function to get the the list of Job Runs
+   * @return
+   */
+  def getJobRunsDF: DataFrame = {
+    val jobsRunsEndpoint = "jobs/runs/list"
+    val jsonQuery = Map(
+      "expand_tasks" -> "true"
+    )
+    ApiCallV2(config.apiEnv,jobsRunsEndpoint, jsonQuery, 2.1)
+      .execute()
+      .asDF()
+      .withColumn("organization_id", lit(config.organizationId))
+  }
+  def getClusterLibraries: DataFrame = {
+    val libsEndpoint = "libraries/all-cluster-statuses"
+    ApiCallV2(config.apiEnv, libsEndpoint)
+      .execute()
+      .asDF()
+      .withColumn("organization_id", lit(config.organizationId))
+  }
+
+  def getClusterPolicies: DataFrame = {
+    val policiesEndpoint = "policies/clusters/list"
+    ApiCallV2(config.apiEnv, policiesEndpoint)
+      .execute()
+      .asDF()
+      .withColumn("organization_id", lit(config.organizationId))
+  }
+
+  def getTokens: DataFrame = {
+    val tokenEndpoint = "token/list"
+    ApiCallV2(config.apiEnv, tokenEndpoint)
+      .execute()
+      .asDF()
+      .withColumn("organization_id", lit(config.organizationId))
+  }
+
+  def getGlobalInitScripts: DataFrame = {
+    val globalInitScEndpoint = "global-init-scripts"
+    ApiCallV2(config.apiEnv, globalInitScEndpoint)
+      .execute()
+      .asDF()
+      .withColumn("organization_id", lit(config.organizationId))
+  }
+
+  /**
+   * TODO - Need to find author of the below code
+   *
+   */
+  //  private def clusterState(apiEnv: ApiEnv): String = {
```

Comment:
Do we need this unused commented code?

Reply:
we need to find the author for this code. Thats why we kept the commented code